Dockerfile.alpine

```diff
@@ -0,0 +1,78 @@
+FROM alpine:3.10
+
+ENV RUBY_VERSION=2.6.3
+ENV FINDSECBUGS_VERSION=1.8.0
+ENV OWASP_VERSION=5.0.0
+
+ARG FINDSECBUGS_FOLDER=/usr/local/opt/findsecbugs
+ARG OWASP_DEP_FOLDER=/usr/local/bin/owaspdependency
+
+RUN apk update  && \
+    apk add  bash \
+    && bash --login
+RUN apk add ca-certificates \
+    openssl openssl-dev \
+    alpine-sdk gcc g++ make \
+    libffi-dev zlib zlib-dev \
+    gnupg procps musl-dev sqlite \
+    linux-headers shadow \
+    nodejs npm yarn \
+    openjdk8-jre maven \
+    python python-dev \
+    perl git \
+    php7 php7-cli \
+    pcre-dev \
+    && curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py" \
+    && python get-pip.py \
+    && pip install safety==1.8.4 piprot==0.9.10 bandit==1.5.1 \
+    && curl -sSL https://github.com/rvm/rvm/tarball/stable -o rvm-stable.tar.gz \
+    && echo 'export rvm_prefix="$HOME"' > /root/.rvmrc \
+    && echo 'export rvm_path="$HOME/.rvm"' >> /root/.rvmrc \
+    && mkdir rvm && cd rvm \
+    && tar --strip-components=1 -xzf ../rvm-stable.tar.gz \
+    && ./install --auto-dotfiles --autolibs=0 \
+    && cd .. \
+    # && rm -rf rvm-stable rvm-stable.tar.gz \
+    $$ source ~/.rvm/scripts/rvm
+RUN /bin/bash -l -c "rvm requirements" \
+    && /bin/bash -l -c "rvm autolibs 4" \
+    && yes | /bin/bash -l -c "rvm install ${RUBY_VERSION}" \
+    && /bin/bash -l -c "gem install bundler:2.0.1 bundler-audit:0.6.1 brakeman:4.5.1" \
+    && /bin/bash -l -c "bundle audit update" \
+    && mkdir -p ${FINDSECBUGS_FOLDER} && cd ${FINDSECBUGS_FOLDER} \
+    && wget --quiet https://github.com/find-sec-bugs/find-sec-bugs/releases/download/version-${FINDSECBUGS_VERSION}/findsecbugs-cli-${FINDSECBUGS_VERSION}.zip \
+    && unzip -q findsecbugs-cli-${FINDSECBUGS_VERSION}.zip \
+    && rm findsecbugs.sh \
+    && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+ENV PATH $HOME/.cargo/bin:$PATH
+
+COPY scripts/findsecbugs.sh ${FINDSECBUGS_FOLDER}/findsecbugs.sh
+RUN chmod +x ${FINDSECBUGS_FOLDER}/findsecbugs.sh \
+    && ln -s ${FINDSECBUGS_FOLDER}/findsecbugs.sh /usr/local/bin/findsecbugs \
+    && mkdir $OWASP_DEP_FOLDER && cd $OWASP_DEP_FOLDER \
+    && wget --quiet http://dl.bintray.com/jeremy-long/owasp/dependency-check-${OWASP_VERSION}-release.zip \
+    && unzip -q dependency-check-${OWASP_VERSION}-release.zip \
+    && chmod +x $OWASP_DEP_FOLDER/dependency-check/bin/dependency-check.sh \
+    && rm dependency-check-${OWASP_VERSION}-release.zip \
+    && mv dependency-check/bin/dependency-check.sh dependency-check/bin/dependency-check \
+    && cd /usr/local/bin \
+    && wget --quiet https://get.sensiolabs.org/security-checker.phar \
+    && chmod +x security-checker.phar && cd \
+    && source $HOME/.cargo/env \
+    && rustc --version \
+    && cargo --version \
+    && cargo install cargo-audit \
+    && cargo audit --version \
+    && mkdir -p /hawkeye
+
+COPY . /hawkeye
+RUN npm install --production --quiet \
```

Comment:
I believe you miss the "cd /hawkeye" command for npm to install. You can check your docker build log. The npm command will warn that there is no package.json file. However, the build will be successful. So I suspect that your docker image will not be able to use.